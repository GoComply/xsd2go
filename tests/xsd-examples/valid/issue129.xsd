<?xml version="1.0" encoding="UTF-8"?>
<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
            elementFormDefault="qualified" attributeFormDefault="qualified" version="1.0">
    <xsd:annotation/>
    <xsd:complexType name="AAA">
        <xsd:annotation/>
        <xsd:sequence>
            <xsd:element name="List" minOccurs="0">
                <xsd:annotation/>
                <xsd:complexType>
                    <xsd:sequence>
                        <xsd:element name="CCC" type="xsd:string" minOccurs="1" maxOccurs="unbounded"/>
                    </xsd:sequence>
                </xsd:complexType>
            </xsd:element>
        </xsd:sequence>
    </xsd:complexType>
    <xsd:complexType name="BBB">
        <xsd:annotation/>
        <xsd:sequence>
            <xsd:element name="List" minOccurs="0">
                <xsd:annotation/>
                <xsd:complexType>
                    <xsd:sequence>
                        <xsd:element name="DDD" type="xsd:int" minOccurs="1" maxOccurs="unbounded"/>
                    </xsd:sequence>
                </xsd:complexType>
            </xsd:element>
        </xsd:sequence>
    </xsd:complexType>
</xsd:schema>