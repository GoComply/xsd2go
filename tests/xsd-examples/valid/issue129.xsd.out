// Code generated by https://github.com/gocomply/xsd2go; DO NOT EDIT.
// Models for issue129
package issue129

import (
	"encoding/xml"
)

// Element
type List struct {
	XMLName xml.Name `xml:"List"`

	Ccc []string `xml:",any"`
}

// Element
type List1 struct {
	XMLName xml.Name `xml:"List"`

	Ddd []int `xml:",any"`
}

// XSD ComplexType declarations

type Aaa struct {
	XMLName xml.Name

	List *List `xml:",any"`
}

type Bbb struct {
	XMLName xml.Name

	List *List `xml:",any"`
}

// XSD SimpleType declarations
